package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "durable-functions",
-    "version": "3.0.0-alpha.5",
+    "version": "3.0.0",

```

Comment:
I just released 4.0.0 of "@azure/functions". Please update the reference below to remove "-alpha.8"